.buildkite/custom-tests.json

```diff
@@ -0,0 +1,14 @@
+{
+  "tests": [
+    {
+      "test_name": "build-mshv",
+      "command": "cargo build --release --no-default-features --features mshv",
+      "platform": ["x86_64"]
+    },
+    {
+      "test_name": "clippy-mshv",
+      "command": "cargo clippy --workspace --bins --examples --benches --no-default-features --features mshv --all-targets -- -D warnings",
```

Comment:
Shouldn't this be:
```cargo clippy --workspace --bins --examples --benches --features=mshv --features mshv --all-targets -- -D warnings``` instead?

Reply:
This crate has defaulted to use `kvm` in `Cargo.toml`. That was introduced to avoid breaking existing users.

So we do need to disable all the default features (`kvm` in this case) and then enable `mshv` explicitly.

The command you wrote runs fine, but because it did not disable `kvm` it would not be testing `mshv` at all.

Reply:
Right! My bad. LGTM!